cyhy/util/util.py

```diff
@@ -18,20 +18,25 @@
     "report_dates",
     "utcnow",
     "warn_and_confirm",
+    "parse_domains",
 ]
 
-import sys
-import itertools
-import json
-import bson
+# Standard Python Libraries
 from collections import OrderedDict
-import netaddr
-import dateutil.tz as tz
 from datetime import datetime
-import time
-import logging
 from logging.handlers import RotatingFileHandler
+import bson
+import itertools
+import json
+import logging
+import sys
+import time
+
+# Third-Party Libraries
 from dateutil.relativedelta import relativedelta, SU, MO, TU, WE, TH, FR, SA
```

Comment:
Just to keep with how `isort` would format this.

```suggestion
from dateutil.relativedelta import FR, MO, SA, SU, TH, TU, WE, relativedelta
```

Reply:
Please see https://github.com/cisagov/cyhy-core/pull/106/commits/fa6b2fa74c9ad5ccc0290c2d54f846beffb569ec.